.dockerignore

```diff
@@ -0,0 +1,18 @@
+storage/
+metadata_data/
+.dockerignore
+# Byte-compiled / optimized / DLL files
+__pycache__/
+**/__pycache__
+**/*__pycache__
+**/*__pycache__*
+**/*__pycache__/
+**/__pycache__/
+*.pyc
+
+database/*
+database
+
+.git
+.git/
+.git/*
```

Comment:
I think we can simplify this file. Here is what we have internally that we can publish:

```
pipeline/out
venv*

.git
node_modules

**/*.py[cod]
**/__pycache__
**/*$py.class
```